### DISCUSSION TITLE
Compliments to mbnuqw

### DISCUSSION THREAD

**Discussion by ghost** [mbnuqw/sidebery#716] [General]
_Greetings!_ This is mainly for @mbnuqw, but for anyone deciding whether to install/use Sidebery or not, as well. I didn't know where to place this, so I chose the **General** category, and I hope this is acceptable. It can be moved, obviously, if necessary.

I merely wanted to proclaim to the world that **Sidebery** is—without a doubt—the **greatest** and most stable, useful, and efficient Tab manager in existence, at least in my experience. And that's actually saying a lot considering my nearly three decades of experience on the Internet (since before most people even knew what the Internet _was_) and with many, many browsers, including _Firefox_ (and my now preferred non-traitorous _Waterfox_ and _Pale Moon_), _Chrome_, Edge, Opera, IE, etc., and several that have sadly or thankfully disappeared (e.g., _Netscape Navigator_, _Prodigy_, etc. 😉).

I am obviously not merely an end-user, but a developer as well, so I can say with confidence that Sidebery is a well-designed add-on with a very attractive UI/UX, and I **highly** recommend it to anyone who wants a finely crafted, quite stable add-on for managing one's tabs and containers, however many there may be (I'm notorious for having too many open). It's a testament to the wisdom, experience and—pardon the term—'leetness of Mr. Badryzlov, and I'm very grateful I was able to happen upon it while looking for a replacement for two others, which are adequate and even awesome in many ways, but didn't suit my needs (namely, _Tree Style Tabs_ and _Tree Tabs_).

Anyway, forgive my "wall of text", novella-length comment. I just want you to know, Mr. Badryzlov, that you have my eternal gratitude. I am and remain your faithful fan and proponent always,

—OGfrostbyte (Seän D. Shepherd)

**Comment by thomascwells** [comment]
Sidebery is a life-changing extension for me. I don't know how I'll ever live without it again. 

I was trying to find a way to send the developer a monetary tip or some other support for their hard work, but this was the best I found. 

Please let us know if we can send you some $$ sometime in the future.
